### DISCUSSION TITLE
Make user query permanent?

### DISCUSSION THREAD

**Discussion by gilgongo** [FreshRSS/FreshRSS#7096] [Q&A]
Loving FreshRSS!

Does anyone know how I can make a user query permament (other than bookmarking it)? So the query is applied by default?

**Comment by Alkarex** [comment]
You can save it an access it from the corresponding menu: 
![image](https://github.com/user-attachments/assets/d10dca29-a3e7-4681-a9c7-6f0e2aef1824)

Or indeed make a browser bookmark of that URL

**Reply by gilgongo** [reply]
Ah yes, but I was wondering if it could be applied by default. I guess I might be able to use an Apache rewrite?

**Comment by math-GH** [comment]
Do you want to use only 1 user query as permanent view?

If you are able to have 2 FreshRSS instances than:
- instance 1: set the query and share it via RSS
- instance 2: import the created query
